extensions/amp-base-carousel/1.0/test/test-amp-base-carousel.js

```diff
@@ -94,161 +93,6 @@ describes.realWin(
       toggleExperiment(win, 'bento-carousel', false, true);
     });
 
-    it('should render slides and arrows when built', async () => {
```

Comment:
Don't we want to leave the old tests in still? So we don't accidently break existing functionality.